src/js/components/Widgets/BallotItemSupportOpposeCountDisplay.jsx

```diff
@@ -619,7 +619,7 @@ class BallotItemSupportOpposeCountDisplay extends Component {
             placement={this.props.popoverBottom ? 'bottom' : 'top'}
             overlay={scoreInYourNetworkPopover}
           >
-            <span className="network-positions-stacked__support-score u-cursor--pointer u-no-break">
+            <NetworkScore>
```

Comment:
Hi @lazorfuzz, I'd love to discuss the changes like this that are moving away from using WebApp/src/sass to using styles embedded in the components. The biggest problem I see is that with styled divs we don't currently have a way to use standardized variables, which we can (and use heavily) within our WebApp/src/sass files. An example is defining our breakpoints in components, like this: @media (max-width: 768px) {